birdhouse/deployment/.gitignore

```diff
@@ -0,0 +1 @@
+cron
```

Comment:
BLOCKING: if we actually need this `.gitignore` here, there is a bug somewhere because this dir do not contain any components.

This one easy to test, just remove it see if you need it.